ci/test/05_before_script.sh

```diff
@@ -6,19 +6,28 @@
 
 export LC_ALL=C.UTF-8
 
-DOCKER_EXEC echo \> \$HOME/.bitcoin  # Make sure default datadir does not exist and is never read by creating a dummy file
+# Make sure default datadir does not exist and is never read by creating a dummy file
+if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+  echo > $HOME/Library/Application\ Support/Bitcoin
+else
+  DOCKER_EXEC echo \> \$HOME/.bitcoin
+fi
 
-mkdir -p depends/SDKs depends/sdk-sources
+if [ "$TRAVIS_OS_NAME" != "osx" ]; then
```

Comment:
Maybe should add a comment explaining this check. I'm confused about why all of this is skipped on osx. Does the depends build not work there? It would seem cleaner to use NO_DEPENDS variable to skip depends steps than to tie the decision to the travis operating system name.

Reply:
Thanks for the review @ryanofsky . This check looks indeed redundant. Removed in #17176